Content.Shared/DrawDepth/DrawDepth.cs

```diff
@@ -9,7 +9,16 @@ public enum DrawDepth
         /// <summary>
         ///     This is for sub-floors, the floors you see after prying off a tile.
         /// </summary>
-        LowFloors = DrawDepthTag.Default - 6,
+        LowFloors = DrawDepthTag.Default - 10,
+
+        // various entity types that require different
+        // draw depths, as to avoid hiding
+        #region SubfloorEntities
+        ThickPipe = DrawDepthTag.Default - 9,
+        ThickWire = DrawDepthTag.Default - 8,
+        ThinPipe = DrawDepthTag.Default - 7,
+        ThinWire = DrawDepthTag.Default - 6,
+        #endregion
```

Comment:
lowfloors should be below the subfloor entities, no?

Reply:
i think lowfloors should be at -10 right now, pretty much below all the other entities